docs/cloud-adoption/operations/overview.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Enterprise Cloud Adoption: Operations overview"
+description: Overview of operations content for Azure enterprise cloud adoption
+author: petertaylor9999
+---
+
+# Enterprise Cloud Adoption: Operations overview
+
+This section of Azure enterprise cloud adoption covers the topic of **operations**. 
+
+Once your Enterprise is engaged in a **digital transformation**, a majority of the work done on the design and implmentation teams will revolve around migrating existing workloads from on-premises to Azure, developing and testing new cloud-native applications in Azure, and incorporating new innovative Azure services into existing on-premises workloads. However, these are just the first step in the digital transformation. Once these workloads are up and running in Azure, the next step is to **operate** them in the cloud.
+
+Operating in the cloud refers to the IT processes and non-functional requirements necessary to run workloads in Azure as a business. This includes monitoring of workloads, analyzing dependencies for bottlenecks, developing strategies for disaster recovery, and more.
```

Comment:
Do you want to add the blue button for the next topic?

Reply:
👍